src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetectionExecutor.swift

```diff
@@ -168,11 +168,28 @@ final class ExposureDetectionExecutor: ExposureDetectionDelegate {
 			writtenPackages: WrittenPackages,
 			completion: @escaping (Result<ENExposureDetectionSummary, Error>) -> Void
 	) -> Progress {
+
+		// Clear the key packages and app config on ENError = 2 = .badParameter
+		// For more details, see: https://jira.itc.sap.com/browse/EXPOSUREAPP-3297
+		func clearCacheOnError2(error: Error) {
```

Comment:
why call this func ..OnError2 and explain the error code in the comment?
I think we should use a better name here, like
clearCacheOnErrorBadParameter

Reply:
:) Renamed!